src/main/protobuf/CommonIndexFileFormat.proto

```diff
@@ -1,25 +1,56 @@
 syntax = "proto3";
+
 package io.osirrc.ciff;
 
+// An index stored in CIFF is a single file comprised of exactly the following:
+//  - A Header protobuf message,
+//  - Exactly the number of PostingsList messages specified in the num_postings_lists field of the Header
+//  - Exactly the number of DocRecord messages specified in the num_doc_records field of the Header
+// The protobuf messages are defined below.
+
+// This is the CIFF header. It always comes first.
 message Header {
-  int32 num_postings_lists = 1;
-  int32 num_docs = 2;
+  int32 version = 1;              // Version.
+
+  int32 num_postings_lists = 2;   // Exactly the number of PostingsList messages that follow the Header.
+  int32 num_doc_records = 3;      // Exactly the number of DocRecord messages that follow the PostingsList messages.
+
+  // The total number of postings lists in the collection; the vocabulary size. This might differ from
+  // num_postings_lists, for example, because we only export the postings lists of query terms.
+  int32 total_postings_lists = 4;
```

Comment:
Would `vocab_size` be more clear?

Reply:
Not in my opinion.  You use num_ and total_ as prefixes and the meanings of each is clear (to me).

Reply:
but you might change the suffixes so that they match - perhaps num_doc_records becomes num_docs

Reply:
renamed to `num_docs` per suggestion.